src/mono/wasm/wasm.proj

```diff
@@ -293,7 +293,6 @@
       <_EmccLinkFlags Include="-s EXPORTED_RUNTIME_METHODS=$(_EmccExportedRuntimeMethods)" />
       <_EmccLinkFlags Include="-s EXPORTED_FUNCTIONS=$(_EmccExportedFunctions)" />
       <_EmccLinkFlags Include="--source-map-base http://example.com" />
-      <_EmccLinkFlags Include="-s STRICT_JS=1" />
```

Comment:
Is this no longer supported by emcc?

Reply:
Not together with `MODULARIZE` https://github.com/emscripten-core/emscripten/blob/main/emcc.py#L1998-L1999